packages/story-editor/src/components/errorBoundary/errorActions.js

```diff
@@ -47,8 +52,9 @@ const Wrapper = styled.div`
 `;
 
 function ErrorActions({ error, errorInfo }) {
-  const body = encodeURIComponent(`${error}${errorInfo.componentStack}`);
-  const reportUrl = `https://github.com/googleforcreators/web-stories-wp/issues/new?labels=Type%3A+Bug&template=bug_report.md&title=${error}&body=${body}`;
+  const textAreaContent = `${error}\n${errorInfo.componentStack}`;
+  const reportUrl =
+    'https://wordpress.org/support/plugin/web-stories/#new-topic-0';
```

Comment:
Should we update the text as well to specify to copy the error / take a screenshot of it for the support topic?

Reply:
Was not sure about it; felt like it wasn't necessary because users usually provide any error message or take a screenshot anyway.

Reply:
> because users usually provide any error message or take a screenshot anyway.

For some reason I've generally had the opposite experience -- the users just say they did X and then something crashed 😛 
But perhaps it's different here if the error is visible like this and not in console only.

Reply:
We'll see how it goes, we can always refine it. I think this is a great improvement already.